WordPress/src/main/java/org/wordpress/android/util/CrashLoggingUtilsWrapper.kt

```diff
@@ -0,0 +1,13 @@
+package org.wordpress.android.util
+
+import dagger.Reusable
+import javax.inject.Inject
+
+@Reusable
+class CrashLoggingUtilsWrapper @Inject constructor() {
+    fun log(message: String?) = CrashLoggingUtils.log(message)
+    fun log(exception: Throwable) = CrashLoggingUtils.log(exception)
+    fun logException(tr: Throwable, tag: AppLog.T) = CrashLoggingUtils.logException(tr, tag)
```

Comment:
I can see that the parameter of type Throwable is called `tr`  in the `CrashLoggingUtils` function. Do we want to leave this as is or should we rename it to `throwable` or something else? Our parameter names have been pretty consistent with the types they represent.

Reply:
Makes sense 👍 Fixed in 2bd574b13e755a27c0ad280ed27305f757841657